src/runner/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+set(LIB_FILES
+        "${FAABRIC_INCLUDE_DIR}/faabric/runner/FaabricMain.h"
+        FaabricMain.cpp
+        )
+
+faabric_lib(runner "${LIB_FILES}")
```

Comment:
I'm not 100% sure whether it works with the `faabric_lib` function, if it was `add_library` I think it would be fine. I'll see if it's a straight swap, if not we can update all uses of `faabric_lib` as part of that issue.